analysis/Scripts/dalitzPlane.py

```diff
@@ -1,84 +1,231 @@
-
 #################
 # Dalitz plot / sPlot
 # Requires sweights script and Imports.py
 #################
 
 
-import ROOT
-from Imports import *
-
-
+import ROOT, sys, getopt, os
+import Imports
 
 # Define what type of plot you want
 dataType = "MC" #dataType = MC (monte carlo), or = data
 particle = "Lc" # valid types :- Xic or Lc (For MC studies)
 # Define if you want to add sWeights
 addsWeights = True
-inputdir = pwd + "4_reduced/"
-outputdir = pwd + "output/"
-
-# Get the data
-if dataType == "data":
-  f = ROOT.TFile.Open(inputdir+"4_Lc_cut_reduced.root", "READONLY")
-  tree = f.Get("DecayTree;48")
-  cuts = "(1==1)"
-elif dataType == "MC": #Todo: check both trees for updates from ganga?
-  cuts = getMCCuts(particle)
-  addsWeights = False  #double check to ensure sWeights never used for MC files
-  if particle == "Xic":
-    Xic_MC_datatree_1()
-    tree = Xic_MC_tree_1 #Todo: check tree 1 or 2 or both?
-  if particle == "Lc":
-    Lc_MC_datatree()
-    tree = Lc_MC_tree
-
-if(addsWeights) :
-  # If we made an sWeight friend tree: add it, and use sWeights. Make sure same cuts (--> #entries) as swTree!
-  wfile = ROOT.TFile.Open("{0}dalitz_temp.root".format(outputdir),"RECREATE")
-  swcuts = "lcplus_MM >= 2240 && lcplus_MM <= 2340 && lcplus_P >= 5000 && lcplus_P <= 200000 && lcplus_TAU >= 0 && lcplus_TAU <= 0.007"
-  tree.Print()
-  cuttree = tree.CopyTree(swcuts)
-  print("cutTree nEvents = {0}".format(cuttree.GetEntries()))
-  cuttree.AddFriend("swTree",pwd+"/output/sWeight_swTree.root")
-  weightvar = "swTree.sw_sig"
-else :
-  cuttree = tree
-  weightvar = "1"
 
+#Input dir is where the reduce tuples are, output is where we will make our plots and our friend trees are in sweightdir
+inputdir = "/dcache/bfys/jtjepkem/binned_files/"
```

Comment:
Should we already make the changes that are compatible with the Imports library that will get merged with the pull #60 ?  This path would be imported as TUPLE_PATH

Reply:
Other than that -which might just be better left for a future pull request- everything seems perfect!